man/create.barplot.Rd

```diff
@@ -386,6 +386,22 @@ create.barplot(
     resolution = 100
     )
 
+# Log-Scaled Axis
+log.data <- data.frame(
+    x = 10 ** sample(1:15, 5),
+    y = LETTERS[1:5]
+    );
+
+create.barplot(
+    x ~ y,
+    log.data,
```

Comment:
Don't mix named and positional arguments, use `formula = ` and `data = ` throughout

Reply:
Sure, looks like this is enforced across all examples. Fixed.

Reply:
> Sure, looks like this is enforced across all examples. Fixed.

Need to update for `create.barplot.Rd`, otherwise looks good!